src/test/scala/cromwell/binding/WdlExpressionToFileSpec.scala

```diff
@@ -0,0 +1,140 @@
+package cromwell.binding
+
+import cromwell.binding.values._
+import cromwell.engine.EngineFunctions
+import org.scalatest.mock.MockitoSugar
+import org.scalatest.{Matchers, FlatSpec}
+
+import scala.util.{Try, Success, Failure}
+
+class WdlExpressionToFileSpec extends FlatSpec with Matchers with MockitoSugar {
+  val expr: String => WdlExpression = WdlExpression.fromString
+  def noLookup(String: String): WdlValue = fail("No identifiers should be looked up in this test")
+
+  val engineFunctions = new MockEngineFunctions
+
+  "anonFiles function" should "find no anonymous files in an empty expression" in {
+    val e: WdlExpression = expr("")
+    e.preevaluateExpressionForFilenames(noLookup, engineFunctions) match {
+      case Success(files) => assert(files isEmpty)
```

Comment:
Warning:(19, 43) postfix operator isEmpty should be enabled
by making the implicit value scala.language.postfixOps visible.
This can be achieved by adding the import clause 'import scala.language.postfixOps'
or by setting the compiler option -language:postfixOps.
See the Scala docs for value scala.language.postfixOps for a discussion
why the feature should be explicitly enabled.
      case Success(files) => assert(files isEmpty)